public/framework/scss/ctm_components/_media.scss

```diff
@@ -33,6 +33,22 @@ video {
   background: $video-bg;
 }
 
+[data-splash-screen] {
```

Comment:
What about the splash screen options? Depending on the image size, the image is displayed oversized withhin an overflow-hidden container. What if we made an `object-fit` for splash-screens adjustable here?

Reply:
> What about the splash screen options? Depending on the image size, the image is displayed oversized withhin an overflow-hidden container. What if we made an `object-fit` for splash-screens adjustable here?

That'd mean it would be a bug already - Using `object-fit: cover` as a default would be preferabble imho.

Reply:
This shouldn't be fixed here but within main